### PR TITLE
Single pipeline for processing NextSeq fastq data

### PR DESCRIPTION
**Pull request by amberlin**
Yup there is a windows exe in there ... supposed to be there .... It might even work

### CODE REVIEW

File: fastq_file_utils/process_nextseq_fastq.py [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+
+    description = "Processes NextSeq data by merging all 4 lanes and optionally subsampling the data"
+
+    # Create instance of ArgumentParser
+    parser = argparse.ArgumentParser(description=description)
+
+    selection = parser.add_mutually_exclusive_group()
+
+    parser.add_argument("-d",
+                        "--directory",
```
Comment (EReckase) [inline fastq_file_utils/process_nextseq_fastq.py]: I hate this one arg per line thing.

**Comment by EReckase** [comment]
looks good
:+1:

File: fastq_file_utils/process_nextseq_fastq.py (hunk 1 above)
Comment (amberlin) [inline fastq_file_utils/process_nextseq_fastq.py]: I'd merge the short and long but otherwise I like them broken out

**Comment by amberlin** [comment]
thanks @EReckase

**Pull request merged**